.github/workflows/benchmarks_run.yml

```diff
@@ -0,0 +1,40 @@
+name: Run and Cache Benchmarks
+
+on:
+  pull_request:
+    types: [ opened, reopened, edited, synchronize ]
```

Comment:
By default pull_request  runs on `[ opened, reopened, synchronize ]`. `edited` applies to changes in the title or body, which doesn't affect this workflow and so doesn't need rerunning. With that, I think we can just leave the defaults and not specify `types` at all?

Reply:
I was curious about that too when I saw it in the bencher docs, but pulling up the definition of the event:
> The title or body of a pull request was edited, or *the base branch of a pull request was changed.*

(emph. mine), seems like a case we'd want to capture.

Reply:
On the other hand, that seems like it will occur much less frequently than normal edits, I'll change this back to the defaults.